37_environment_lab/envlab.txt

```diff
@@ -1,15 +1,50 @@
 1. Write the following aliases into the file aliases (and upload that file too):
-  - ll => ls -l
-  - rm => rm -i
-  - cp => cp -i
+
+gedit someAlias.bash
+bash someAlias.bash
```

Comment:
From bash it's better to run `source someAlias.bash`, as that injects the aliases into your running shell